data/brands/amenity/recycling.json

```diff
@@ -10,6 +10,42 @@
     }
   },
   "items": [
+    {
+      "displayName": "BottleDrop",
+      "id": "bottledrop-f8d7ba",
+      "locationSet": {"include": ["us"]},
+      "matchNames": ["bottle drop"],
+      "tags": {
+        "amenity": "recycling",
+        "brand": "BottleDrop",
+        "brand:wikidata": "Q107673127",
+        "name": "BottleDrop",
+        "note": "BottleDrop is the self service station. For bag drop, use BottleDrop Express instead.",
+        "operator": "Oregon Beverage Recycling Cooperative",
+        "recycling:cans": "yes",
+        "recycling:glass_bottles": "yes",
+        "recycling:plastic_bottles": "yes",
+        "recycling_type": "centre"
+      }
+    },
+    {
+      "displayName": "BottleDrop Express",
+      "id": "bottledropexpress-f8d7ba",
+      "locationSet": {"include": ["us"]},
+      "matchNames": ["bottle drop"],
+      "tags": {
+        "amenity": "recycling",
+        "brand": "BottleDrop",
+        "brand:wikidata": "Q107673127",
+        "name": "BottleDrop Express",
+        "note": "BottleDrop Express is the bag drop service. For self service, use BottleDrop instead.",
```

Comment:
@bhousel is a note how we handle messages to the user in editors?

Reply:
No the `note` isn't shown anywhere else - it's just a note for the name-suggestion-index team..